docs/4.3/aws_oss_guide.md

```diff
@@ -388,12 +381,311 @@ To upgrade to a newer version of Teleport:
+
+```bash
+export TELEPORT_PUBLIC_DNS_NAME="[teleport-proxy-url]"
+export GH_CLIENT_ID="[github-client-id]"
+export GH_SECRET="[github-oauth-secret]"
+export GH_ORG="[github-org]"
+export GH_TEAM="[github-team]"
+
+cat > github.yaml << EOF
+kind: github
+version: v3
+metadata:
+  # connector name that will be used with 'tsh --auth=github login'
+  name: github
+spec:
+  # client ID of Github OAuth app
+  client_id: $GH_CLIENT_ID
+  # client secret of Github OAuth app
+  client_secret: $GH_SECRET
+  # connector display name that will be shown on web UI login screen
+  display: Github
+  # callback URL that will be called after successful authentication
+  redirect_url: https://$TELEPORT_PUBLIC_DNS_NAME:3080/v1/webapi/github/callback
+  # mapping of org/team memberships onto allowed logins and roles
+  teams_to_logins:
+  - kubernetes_groups:
+  # change this to restrict the access of users logging in via GitHub;
+  # system:masters gives everyone full access to the entire cluster
+    - system:masters
+    logins:
+    - github
+    - ec2-user
+    organization: $GH_ORG
+    team: $GH_TEAM
+EOF
+```
+Use `tctl` to create the github auth connector.
+
+```bash
+sudo /usr/local/bin/tctl create -f ./github.yaml
+```
+
+#### Testing Teleport & EKS Setup
+
+
+```
+➜  ~ tsh login --proxy=[teleport-proxy-url].practice.io:3080
+If browser window does not open automatically, open it by clicking on the link:
+ http://127.0.0.1:64467/54e5d06a-c509-4077-bf54-fb27fd1b8d50
+> Profile URL:  https://[teleport-proxy-url].practice.io:3080
+  Logged in as: benarent
+  Cluster:      teleport-eks
+  Roles:        admin*
+  Logins:       github, ec2-user
+  Valid until:  2020-06-30 02:12:54 -0700 PDT [valid for 12h0m0s]
+  Extensions:   permit-agent-forwarding, permit-port-forwarding, permit-pty
+
+
+* RBAC is only available in Teleport Enterprise
+  https://gravitational.com/teleport/docs/enterprise
+```
+
+On your local machine test using `kubectl get pods --all-namespaces`
+
+```bash
+➜  ~ kubectl get pods --all-namespaces
+NAMESPACE     NAME                       READY   STATUS    RESTARTS   AGE
+kube-system   aws-node-56p6g             1/1     Running   0          5h28m
+kube-system   aws-node-7dv5j             1/1     Running   0          5h28m
+kube-system   coredns-5c97f79574-69m6k   1/1     Running   0          5h36m
+kube-system   coredns-5c97f79574-dq54w   1/1     Running   0          5h36m
+kube-system   kube-proxy-7w4z4           1/1     Running   0          5h28m
+kube-system   kube-proxy-c5nv2           1/1     Running   0          5h28m
+```
+
+## Running Teleport Enterprise on AWS
+Most of this guide has been designed for OSS Teleport. Most of this guide also applies to Teleport Enterprise with a few extra notes around adding a license and getting the correct binary. If you would like help setting up Teleport Enterprise on AWS, please mail us at [info@gravitational.com](mailto:info@gravitational.com)
```

Comment:
Don't we have a separate guide now for using Terraform with Teleport Enterprise?  Wondering if we should link to that as well. The wording here is kinda sketchy as customers running Teleport Enterprise on AWS is probably a large chunk of our customer base but we aren't providing a good documentation experience for them.

Reply:
Good point, I've added a link to the terraform guide and also the Enterprise Quickstart.